OloEngine/src/OloEngine/Scripting/C#/ScriptGlue.cpp

```
@@ -1044,6 +1044,90 @@ namespace OloEngine
         GetEntity(entityID).GetComponent<ParticleSystemComponent>().System.Emitter.RateOverTime = *v;
     }
 
+    // --- ParticleSystemComponent Wind ---
+
+    static void ParticleSystemComponent_GetWindInfluence(UUID entityID, f32* out)
+    {
+        *out = GetEntity(entityID).GetComponent<ParticleSystemComponent>().System.WindInfluence;
+    }
+
+    static void ParticleSystemComponent_SetWindInfluence(UUID entityID, f32 const* v)
```

Clang (14.0.6)

[warning] parameter name 'v' is too short, expected at least 3 characters

(readability-identifier-length)